tox.ini

```diff
@@ -11,6 +11,8 @@ deps =
     nose
     unittest2
     protobuf==3.0.0-alpha-1
+setenv =
+    PYTHONPATH = {toxinidir}/_testing
```

Comment:
We should be jamming `_testing` into the `PYTHONPATH` IFF `grpc` cannot be imported.  Even better would be to *append* it to `sys.path` (entries in `PYTHONPATH` are prepended).